### ISSUE TITLE
Incorrect description of behavior in docs

### ISSUE THREAD

**Issue by kjarmicki** [schibsted/sdk-js#57]
At http://techdocs.spid.no/sdks/js/events/ it states:

```
SPiD.login  
(...)
Page loads, user is already logged in
```

This is incorrect, `login` event is fired once - when user actually logs in, not if the session is already stored in cache (and valid).

This can be approached in two ways:
- change the description in the docs to fit the behavior and provide another event that's fired when session cache is already populated
- fix login event to match behavior described in docs

**Comment by danielbentes** [comment]
Backwards compatibility would state that the event should be triggered whenever a user is logged in, from cached state or not.

**Comment by marek-nogiec** [comment]
@danielbentes this would mean that it's gonna be fired more often that some scripts may expect (at least the ones that assume it's fired only once) and IMO it doesn't relate well to the event's name which suggests that it's fired after *login* (so *not* after restoring session from cache). I would suggest creating a separate event that triggers both on session init and session restore.

**Comment by danielbentes** [comment]
All I'm saying is that this was triggered on every page refresh (after the script was initialised and the user was present) before (previous versions). Which is why it was documented this way.

I agree that we could change this, but that would mean a breaking change in behaviour. In this case, a better suited event should be triggered and the documentation updated.

**Comment by kjarmicki** [comment]
Ok, so the question really is: is someone already depends on undocumented behavior of `login` event. If we can't know (too many users) or we're certain that someone does depend on it, we should [not break current `login` behavior](https://lkml.org/lkml/2012/12/23/75) and update the docs to state the truth.

If however nobody depends on that, then I'd fix `login` event to behave as it is stated in docs.

From my side, I can confirm that none of the SNT papers depend on it.
@danielbentes do you have some data regarding other SPiD clients?

**Comment by danielbentes** [comment]
I don't have data about other clients. But in any case, this should be announced as a braking change for the version with this fixed.

**Comment by kjarmicki** [comment]
Sure, that's what semver could be used for - bump the minor version and then communicate through appropriate channels about the change (or fix, rather) :)

**Comment by kjarmicki** [comment]
Are you ok with such solution @danielbentes or should we do it other way?

**Comment by danielbentes** [comment]
I'm ok with that. Just remember to update the documentation and prepare a writeup explaining the breaking changes so we can announce it

**Comment by kjarmicki** [comment]
update: for now we're putting it on hold due to more urgent issues
